src/algebra/ordered_group.lean

```diff
@@ -388,17 +399,21 @@ begin
     exact ⟨_, rfl, add_le_add_left' h⟩, }
 end
 
-@[simp] lemma coe_zero [add_monoid α] : ((0 : α) : with_bot α) = 0 := rfl
+@[norm_cast] lemma coe_zero [add_monoid α] : ((0 : α) : with_bot α) = 0 := rfl
+
+@[norm_cast] lemma coe_eq_zero {α : Type*}
```

Comment:
As above, the proof is already `norm_cast`.

Reply:
In fact `norm_cast` can even prove `coe_zero` -- it uses `with_top` proofs to prove `with_bot` theorems, presumably because the `with_bot` instances are defined using `with_top` instances rather than being redefined. I don't know if this is good or bad.